src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.finrem.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+public class AmendedConsentOrder {
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
```

Comment:
Could we use a constant here to represent Date format for all date fields?

Reply:
I don't know, but I am sure we'll have to change a few things when the actual serialisation is tried.

Reply:
Actually we can